### PR TITLE
A fresh popularities.json export

### LINKED ISSUE #1238: A fresh popularities.json export
The current [`popularities.json`](https://github.com/mdn/yari/blob/42dd24906b9f009fcff8cd02c3e7a5ff5933a818/content/files/popularities.json) file was created by me making a Google Analytics 360 export on all pageviews. You have to wait a day for the CSV file to be ready but then you get a massive .csv file. You can then use `import/popularities.js` to generate (from the .csv) a new `popularities.json` file. 

Note. This new file will need to be checked in to Content, not Yari.

### PR DESCRIPTION
**Pull request by peterbe**
Fixes #1238

### CODE REVIEW

**Comment by peterbe** [comment]
@escattone Here's how I tackled it. First of all, I wrote [the instructions](https://github.com/mdn/yari/issues/1238#issuecomment-690685306) for how to do this again. 
Also, I had to refactor the `import/popularities.js` and `import/cli.js` to make it work. The `popularities.json` this created is currently on my disk in the Content folder which I haven't pushed to GitHub yet. But will soon. 

The other thing I did, in the refactor, was that significantly improved the algorithm of parsing the pageview URIs. For example, now it can only include URIs whose `$locale` is in `VALID_LOCALES` in `content/constants.js`. And for URIs (in the CSV) that were typed with the wrong case, I just fixed them. 

My MySQL prod snapshot is from July 31st so the popularities.json is pretty decent. At least for a while. We might want to return to https://github.com/mdn/yari/issues/1238 and re-open it days before we do the final migration. Either that or we just let it be as it is from July 31st and instead set our sights on https://github.com/mdn/yari/issues/1239

**Comment by peterbe** [comment]
I'm not seeking a deep review here. Maybe I'm just reaching out for a bit of check-in. I'm pretty sure it works since the `popularities.json` it produces looks very sane. 
But I'm not blocked because I can soon do the first and initial checkin of Content.

File: import/README.md [hunk 1]
```diff
@@ -4,15 +4,41 @@ This code is a bit rough but it's ultimately just meant to be used one
 single time in its life existence. Once we've run it and checked it in
 and moved on with our post-Wiki life, we can burn this.
 
-## Example run
+## import
 
 You can set `CONTENT_ROOT` and `CONTENT_ARCHIVE_ROOT` to any folder
-but it's **important that directories exists** otherwise Yari will
+but **it's important that directories exists** otherwise Yari will
```
Comment (escattone) [inline import/README.md:10]: Typo:
```suggestion
but **it's important that the directories exist** otherwise Yari will
```

**Review by escattone — APPROVED** [review]
@peterbe Thanks for documenting what you need to do to repeat this task.

**Pull request merged**